content/source/layouts/enterprise2.erb

```diff
@@ -261,6 +261,12 @@
           <li<%= sidebar_current("docs-enterprise2-private-reliability-availability") %>>
             <a href="/docs/enterprise/private/reliability-availability.html">Reliability &amp; Availability</a>
           </li>
+          <li<%= sidebar_current("docs-enterprise2-private-data-security") %>>
+            <a href="/docs/enterprise/private/data-security.html">Data Security</a>
+          </li>
+          <li<%= sidebar_current("docs-enterprise2-private-automated-recovery") %>>
+            <a href="/docs/enterprise/private/automated-recovery.html">Automated Recovery (Installer)</a>
```

Comment:
There's already an automated recovery link in the "Installation (Installer)" section, so we don't need an extra nav link here.